Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Raw_image_dialog.cpp

```diff
@@ -1,19 +1,27 @@
 #include "Raw_image_dialog.h"
+#include <QPushButton>
 
 Raw_image_dialog::Raw_image_dialog(QWidget* parent)
-  : QDialog(parent) 
+  : QDialog(parent)
 {
   setupUi(this);
 }
 
 void Raw_image_dialog::update_image_size() {
-  label_image_size->setNum((int)image_word_size() *
-			   dim_x->value() *
-			   dim_y->value() *
-			   dim_z->value());
+  int size = (int)image_word_size() *
+      dim_x->value() *
+      dim_y->value() *
+      dim_z->value() +
+      offset->value();
+  label_image_size->setText(QString("%1 B").arg(size));
+
+  if(label_image_size->text() == label_file_size->text())
+    buttonBox->button(QDialogButtonBox::Open)->setEnabled(true);
+  else
+    buttonBox->button(QDialogButtonBox::Open)->setEnabled(false);
```

Comment:
Maybe add a tooltip, when the button is disabled, that explains why. And remove the tooltip once is it enabled.